src/main/java/org/aist/aide/finitehorizonservice/domain/models/NewCase.java

```diff
@@ -0,0 +1,17 @@
+package org.aist.aide.finitehorizonservice.domain.models;
+
+import javax.validation.constraints.NotBlank;
+
+public class NewCase {
```

Comment:
Having "New" in this sounds weird and can be confusing. I would just make this case or something more explicit.

Reply:
case is reserved in java... I can make it additionalCase?